feat: php docker page add extra callouts (#5724)
newrelic/docs-website: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx

You can install the [PHP agent](/docs/agents/php-agent/getting-started/introduction-new-relic-php) on a Docker container or other container to monitor one or more of your PHP applications. This is supported for containers that meet [PHP agent requirements](/docs/agents/php-agent/getting-started/php-agent-compatibility-requirements).

<Callout variant="caution">

>  The daemon only transmits data to Newrelic periodically, once every minute. If you are standing up and tearing down containers often, ensure that you leave the daemon container running for enough time to transmit any remaining data.

Reviewer: I think you may mean "starting up" not "standing up"?

>  The daemon only transmits data to Newrelic periodically, once every minute. If you are standing up and tearing down containers often, ensure that you leave the daemon container running for enough time to transmit any remaining data.

Reviewer 2: please use: `New Relic` :)

Wording suggestion: The PHP Agent's daemon transmits data to New Relic periodically during the minute-long [harvest cycle](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/getting-started/glossary#harvest-cycle). If you are standing up and tearing down containers often, ensure that you leave the daemon container running long enough to transmit any remaining data.

(btw it's described in this page: https://docs.newrelic.com/docs/apm/transactions/transaction-traces/introduction-transaction-traces/)

>  The PHP Agent's daemon transmits data to New Relic periodically during the minute-long [harvest cycle](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/getting-started/glossary#harvest-cycle). If you are starting up and tearing down containers often, ensure that you leave the daemon container running long enough to transmit any remaining data.

Reviewer 3: ```suggestion
 The PHP Agent's daemon transmits data to New Relic periodically during the minute-long [harvest cycle](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/getting-started/glossary#harvest-cycle). If you're starting up and tearing down containers often, ensure that you leave the daemon container running long enough to transmit any remaining data.
```

We generally want to use contractions.

* Meet [PHP agent requirements](/docs/agents/php-agent/getting-started/php-agent-compatibility-requirements)
* PHP agent version 9.2 or higher

> To see an example application, go to [New Relic's Explorers Hub](https://discuss.newrelic.com/t/relic-solution-php-agent-and-daemon-containers/84841). Using a separate container for the daemon is recommended if there will be short-lived application containers.

Reviewer 2: suggestion: `If using short-lived application containers, it is recommended to use a separate container for the PHP Agent daemon.`

> To see an example application, go to [New Relic's Explorers Hub](https://discuss.newrelic.com/t/relic-solution-php-agent-and-daemon-containers/84841). If using short-lived application containers, it is recommended to use a separate container for the PHP Agent's daemon.

Reviewer 3: ```suggestion
To see an example application, go to [New Relic's Explorers Hub](https://discuss.newrelic.com/t/relic-solution-php-agent-and-daemon-containers/84841). If you're using short-lived application containers, we recommend you use a separate container for the PHP Agent's daemon.
```

## Install agent and daemon in the same container [#install-same-container]

<Callout variant="caution">

> By default, the first transaction causes the agent to trigger a daemon start and an application connection initialization. However, for performance reasons, the agent does not wait for those operations to complete before it initializes the connection, which can result in the loss of the first few transactions after a container starts. To prevent this, set both `newrelic.daemon.start_timeout` and `newrelic.daemon.app_connect_timeout` entries in the `newrelic.ini` file to recommended values.

Reviewer: Maybe you could add a link to the recommended values?

> By default, the first transaction causes the agent to trigger a daemon start and an application connection initialization. However, for performance reasons, the agent does not wait for those operations to complete before it initializes the connection, which can result in the loss of the first few transactions after a container starts. To prevent this, set both `newrelic.daemon.start_timeout` and `newrelic.daemon.app_connect_timeout` entries in the `newrelic.ini` file to recommended values.

Reviewer 2: what are the recommended values?

> By default, the first transaction causes the agent to trigger a daemon start and an application connection initialization. However, for performance reasons, the agent does not wait for those operations to complete before it initializes the connection, which can result in the loss of the first few transactions after a container starts. To prevent this, set both `newrelic.daemon.start_timeout` and `newrelic.daemon.app_connect_timeout` entries in the `newrelic.ini` file to (recommended values)[https://discuss.newrelic.com/t/php-troubleshooting-framework-configuration/119980] of 5s and 15s respectively.

Reviewer 3: ```suggestion
By default, the first transaction causes the agent to trigger a daemon start and an application connection initialization. For performance reasons, the agent does not wait for those operations to complete before it initializes the connection. This can result in the loss of the first few transactions after a container starts. To prevent this loss, set both `newrelic.daemon.start_timeout` and `newrelic.daemon.app_connect_timeout` entries in the `newrelic.ini` file to our (recommended values)[https://discuss.newrelic.com/t/php-troubleshooting-framework-configuration/119980] of 5s and 15s respectively.
```

Cleaning up a bit of the wording here.

    * `YOUR_LICENSE_KEY`: Replace this with your [New Relic license key](/docs/accounts/install-new-relic/account-setup/license-key). Note that `"REPLACE_WITH_REAL_KEY"` is an actual string in the default **newrelic.ini** file for the PHP agent. Don't edit that string. The `sed` command replaces that default string with the actual 40-character key, in quotes.
    * `YOUR_APPLICATION_NAME`: Replace with the your application name, in quotes.
  </Collapser>
</CollapserGroup>

<Callout variant="info">

>   Additionally docker troubleshooting resources:

Reviewer 3: ```suggestion
  Here are some Docker troubleshooting resources:
```
